### ISSUE TITLE
Feature request: git commit hash in in btrbk --version output

### ISSUE THREAD

**Issue by ian-kelling** [digint/btrbk#321]
At least for for non-release version. I have a script to do backups,
and I want to check that btrbk is the same version or do a make
install if not. This is a minor issue.

**Comment by digint** [comment]
I don't think this is not easily doable. Well I could call `git rev-parse --short HEAD` when starting btrbk, but this seems wrong to me. Please tell me if you have any ideas how this can be easily achieved.
